src/Umbraco.Web.BackOffice/Umbraco.Web.BackOffice.csproj

```diff
@@ -1,11 +1,13 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Unless your plan is to have 3x executable websites but 2 of them (Website and Backoffice) are also used like class libraries when hosting both parts on the same website? In which case things could get a bit weird since we won't know where to put assets/views. 
I think we should probably have netcore3.1 class libs for the back office and frontend and then also have web executables for both the backoffice, frontend, and combined. 
For now, this is a super easy change, i'll change these over to class libs